blog/README.md

```diff
@@ -0,0 +1,83 @@
+# Knative blog
+
+The Knative blog is owned by the [Documentation working group](https://knative.dev/community/contributing/working-groups/working-groups/#documentation) and run by the [Editorial Team](#leadership).
+
+This section covers documentation, processes, and roles for the [Knative blog](https://knative.dev/blog/).
+
+
+## Leadership -- TBD - JUST A PROPOSAL FOR NOW
+
+- **Technical Editors:** [Lionel Villard](https://github.com/lionelvillard), [Evan Anderson](https://github.com/evankanderson)
+- **Copy Editors:** [Ashleigh Brennan](https://github.com/abrennan89), [Caroline Lee](https://github.com/carieshmarie), [María Cruz](https://github.com/macruzbar)
+- **Blog Community Managers:**  [María Cruz](https://github.com/macruzbar), [Jonatas Baldin](https://github.com/jonatasbaldin)
+
+## Contact
+
+- Slack: [#docs](https://knative.slack.com/archives/C9CV04DNJ)
+- Open Community Issues/PRs: Link TBD
+```diff
+- Ideally we set up a query that brings up open PRs associated to the blog
+```
+
+## Submit a Post
+
+Anyone can write a blog post and submit it for review.
+```diff
+- We have to decide here if we allow commercial blogposts or not. It would also be useful if we stablish a difference between that and vendor-specific how-to blogs
+```
+
+To submit a blog post, follow the steps below.
+
+1. [Sign the Contributor License Agreements](https://github.com/knative/community/blob/master/CONTRIBUTING.md#contributor-license-agreements) if you have not yet done so.
+1. Familiarize yourself with the Markdown format for existing blog posts in the [docs repository](https://github.com/knative/docs/tree/master/blog). Blog posts are categorized into different directories. You can explore the directories to find examples.
+1. Write your blog post in a text editor of your choice.
+1. Choose a directory in the [docs repository](https://github.com/knative/docs/tree/master/blog), and click **Create new file**.
+1. Paste your content into the editor and save it. Name the file in the following way: *[BLOG] Your proposed title* , but don’t put the date in the file name. The blog reviewers will work with you on the final file name, and the date on which the blog will be published.
+1. When you save the file, GitHub will walk you through the pull request (PR) process.
+1. A reviewer is assigned to all pull requests automatically. The reviewer checks your submission, and works with you on feedback and final details. When the pull request is approved, the blog will be scheduled for publication.
+
+### Blog Guidelines
+
+Requested content:
+
+```diff
+- Ideally, each of these links to a pull request with the specific blog template, see demo for example
+```
+
+- Knative new feature releases and project updates
+- Tutorials and demos [start a blog](https://github.com/knative/docs/pull/2511)
+- Use cases
```

Comment:
Should this be "user experiences" or "stories from production"?

Reply:
I'd say "Use Cases" is a well known term in the industry, so people relate easily :)